### PR TITLE
[Alluxio-1938] Add unit tests for URIStatus

### PR DESCRIPTION
**Pull request by xgong**
[Alluxio-1938] Add unit tests for URIStatus
https://alluxio.atlassian.net/browse/ALLUXIO-1938

### CODE REVIEW

File: core/client/src/test/java/alluxio/client/file/URIStatusTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    Assert.assertEquals(uriStatus.isPersisted(), fileInfo.isPersisted());
+    Assert.assertEquals(uriStatus.isPinned(), fileInfo.isPinned());
+    Assert.assertEquals(uriStatus.isMountPoint(), fileInfo.isMountPoint());
+    Assert.assertEquals(uriStatus.getFileBlockInfos(),
+        fileInfo.getFileBlockInfos());
+    Assert.assertEquals(uriStatus.toString(), fileInfo.toString());
+  }
+
+  @Test (timeout = 10000)
+  public void testEquals() throws Exception {
```
Comment (aaudiber) [inline core/client/src/test/java/alluxio/client/file/URIStatusTest.java]: Take a look at `CommonTestUtils#testEquals` - it should be useful here.

This method should be called `equalsTest` (we end all our tests with "Test")

File: core/client/src/test/java/alluxio/client/file/URIStatusTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      Assert.assertTrue(e instanceof NullPointerException);
+      Assert.assertTrue(e.getMessage().contains(
+          "Cannot create a URIStatus from a null FileInfo"));
+    }
+  }
+
+  /**
+   * Tests getting and setting fields.
+   */
+  @Test (timeout = 10000)
```
Comment (aaudiber) [inline core/client/src/test/java/alluxio/client/file/URIStatusTest.java]: I don't think these tests need timeouts, nothing should be blocking

**Comment by aaudiber** [comment]
Thanks for the pull request @xgong! I've left a couple of comments on the code. Please also add a link to the JIRA ticket in the pull request description

File: core/client/src/test/java/alluxio/client/file/URIStatusTest.java (hunk 1 above)
Comment (xgong) [inline core/client/src/test/java/alluxio/client/file/URIStatusTest.java]: Thanks for the review. @aaudiber 
Looks like that it is required that the given class has a no-arg constructor to use CommonTestUtils#testEquals. But we do not have no-arg constructor for URIStatus

File: core/client/src/test/java/alluxio/client/file/URIStatusTest.java (hunk 2 above)
Comment (xgong) [inline core/client/src/test/java/alluxio/client/file/URIStatusTest.java]: Sure. Remove the timeouts from the test

File: core/client/src/test/java/alluxio/client/file/URIStatusTest.java (hunk 1 above)
Comment (aaudiber) [inline core/client/src/test/java/alluxio/client/file/URIStatusTest.java]: Ah, you're right, we can leave it how it is then.

**Comment by aaudiber** [comment]
LGTM

**Pull request merged**